reference.md

```diff
@@ -0,0 +1,1393 @@
+# Snapcraft CLI reference
+
+## lifecycle and misc
+
+### version
+
+    $ snapcraft --version
+    2.12.1
+
+### help
+
+    $ snapcraft help
+    Usage:
+      snapcraft [options] [--enable-geoip --no-parallel-build]
```

Comment:
I think --enable-geoip and --noparallel-build shouldn't be here. They are already in the pull and build entries.